specification/cognitiveservices/data-plane/Language/preview/2022-03-01-preview/textanalytics.json

```diff
@@ -645,7 +682,7 @@
     "SingleClassificationDocumentResult": {
       "type": "object",
       "properties": {
-        "classification": {
+        "class": {
```

Comment:
My only concern is what various clients will call this (since `class` is often a reserved/keyword). I don't think I thought about this in the REST API discussion. But we should be able to specify `x-ms-client-name` to rename/quantify the name in client libraries.